x-pack/plugins/cases/server/services/user_actions/index.ts

```diff
@@ -33,30 +47,39 @@ interface PostCaseUserActionArgs extends ClientArgs {
 export class CaseUserActionService {
   constructor(private readonly log: Logger) {}
 
-  public async getAll({ unsecuredSavedObjectsClient, caseId, subCaseId }: GetCaseUserActionArgs) {
+  public async getAll({
+    unsecuredSavedObjectsClient,
+    caseId,
+    subCaseId,
+  }: GetCaseUserActionArgs): Promise<SavedObjectsFindResponse<CaseUserActionResponse>> {
     try {
       const id = subCaseId ?? caseId;
       const type = subCaseId ? SUB_CASE_SAVED_OBJECT : CASE_SAVED_OBJECT;
 
-      return await unsecuredSavedObjectsClient.find<CaseUserActionAttributes>({
+      const userActions = await unsecuredSavedObjectsClient.find<CaseUserActionAttributes>({
         type: CASE_USER_ACTION_SAVED_OBJECT,
         hasReference: { type, id },
         page: 1,
         perPage: MAX_DOCS_PER_PAGE,
         sortField: 'action_at',
         sortOrder: 'asc',
       });
+
+      return transformFindResponseToExternalModel(userActions);
     } catch (error) {
       this.log.error(`Error on GET case user action case id: ${caseId}: ${error}`);
       throw error;
     }
   }
 
-  public async bulkCreate({ unsecuredSavedObjectsClient, actions }: PostCaseUserActionArgs) {
+  public async bulkCreate({
+    unsecuredSavedObjectsClient,
+    actions,
+  }: PostCaseUserActionArgs): Promise<void> {
     try {
       this.log.debug(`Attempting to POST a new case user action`);
 
-      return await unsecuredSavedObjectsClient.bulkCreate<CaseUserActionAttributes>(
+      await unsecuredSavedObjectsClient.bulkCreate<CaseUserActionAttributes>(
```

Comment:
No callers used the result and if we don't return we can avoid having to transform it again so let's not return it for now.